hack/generate-yamls.sh

```diff
@@ -49,6 +49,9 @@ rm -fr ${YAML_OUTPUT_DIR}/*.yaml
 
 # Generated Knative component YAML files
 readonly SERVING_YAML=${YAML_OUTPUT_DIR}/serving.yaml
+readonly SERVING_BETA_YAML=${YAML_OUTPUT_DIR}/serving-beta.yaml
```

Comment:
You're not creating or using this file anywhere.

Reply:
leftover from an edit I backed out, reverted.  good catch.